homework-3/src/components/Forms/FormRestorePassword.js

```diff
@@ -1,19 +1,18 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { Formik } from 'formik';
-import TitleForm from '../common/Titles/StylTitleForm';
-import FormInput from '../FormInput';
-import FormSign from '../common/partsOfTheForm/StylFormSign';
-import BtnSubmitFormReset from './../BtnsComponents/BtnSubmitFormReset';
-import TitleFormArrow from '../TitleFormArrow';
-import InformationText from '../common/StylInformationText';
-import InformationTextForm from '../InformationTextForm';
+import { StylTitleForm } from '../common/Titles/';
+import { StylFormSign } from '../common/partsOfTheForm/';
+import { BtnSubmitFormReset } from './../BtnsComponents/';
+import { FormInput, InformationTextForm, TitleFormArrow } from '../';
+import { StylBoxViewContent } from '../common/Boxes/';
+import { StylInformationText, StylLinkPage } from '../common/';
 import emailImg from './../../img/icons/icon-email.svg';
-import LinkPage from '../common/StylLinkPage';
-import BoxViewContent from '../common/Boxes/StylBoxViewContent';
 
 const FormRestorePassword = () => {
-  let [isEmail, setEmail] = React.useState(false);
-  let [isHiddenForm, setHiddenForm] = React.useState(true);
+  const [isEmail, setEmail] = useState(false),
+    [isHiddenForm, setHiddenForm] = useState(true),
+    regulEmail =
```

Comment:
As i told you before, move this into constats.ts, and import into every component, as the constat.

Reply:
oh, I misunderstood your idea
Fixed